choreolib/src/main/java/choreo/auto/AutoChooser.java

```diff
@@ -108,9 +135,40 @@ public void addAutoRoutine(String name, AutoRoutineGenerator generator) {
   }
 
   /**
-   * Get the currently selected auto routine.
+   * Get the currently selected {@link AutoRoutine}.
+   *
+   * <h3>Recommended Usage</h3>
+   *
+   * Scheduling it as a command.
+   *
+   * <pre><code>
```

Comment:
Do we want to share the RobotModeTriggers approach here?

Reply:
Im sorry, idk what you mean by that.